numpy/ma/core.py

```diff
@@ -82,6 +82,9 @@
 MaskType = np.bool_
 nomask = MaskType(0)
 
+class MaskedArrayFutureWarning(FutureWarning):
```

Comment:
This is to make the message always show? If so, it won't work because derived classes of `FutureWarning` will also be ignored if `FutureWarnins` is.
```
In [1]: import warnings

In [2]: warnings.simplefilter('always')

In [3]: warnings.simplefilter('ignore', FutureWarning)

In [4]: class MyWarn(FutureWarning):   
    pass
   ...: 

In [5]: warnings.warn('hi', FutureWarning)

In [6]: warnings.warn('hi', MyWarn)

In [7]: warnings.warn('hi', RuntimeWarning)
/home/charris/.local/bin/ipython:1: RuntimeWarning: hi
  #!/usr/bin/python
```